docs/argocd-app-of-apps.md

```diff
@@ -163,7 +163,7 @@ Once we are ready to fully remove it we replace the components template with a `
 
 ```yaml
 {{ if .Values.component.enabled }}
-{{ fail("component is DEPRECATED, use "improved solution" instead") }}
+{{ fail "component is DEPRECATED, use "improved solution" instead" }}
```

Comment:
```suggestion
{{ fail "component is DEPRECATED, use improved solution instead" }}
```
The double quotes around `improved solution` are going to cause issues at some point. :D